### PR TITLE
add router-perf and node-density-heavy workload to perfscale ci

### PR DESCRIPTION
**Pull request by qiliRedHat**

### CODE REVIEW

**Comment by qiliRedHat** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-ocp-qe-perfscale-ci-tests

**Comment by qiliRedHat** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-ocp-qe-perfscale-ci-tests

**Comment by qiliRedHat** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-ocp-qe-perfscale-ci-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-ocp-qe-perfscale-ci-tests

**Comment by qiliRedHat** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-ocp-qe-perfscale-ci-tests

**Comment by qiliRedHat** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-ocp-qe-perfscale-ci-tests

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main.yaml [hunk 1]
```diff
@@ -33,6 +33,18 @@ tests:
     - chain: ipi-gcp-pre
     - chain: workers-scale
     workflow: openshift-qe-ocp-qe-perfscale-ci-tests
+- as: ocp-qe-perfscale-router-perf-gcps
```
Comment (paigerube14) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main.yaml:36]: does the gcps at the end stand for anything here?

Comment (qiliRedHat) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main.yaml]: This is a typo for gcp, I did another update. I tried to add router-perf test with different providers in the config.

**Comment by qiliRedHat** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-ocp-qe-perfscale-router-perf-gcp

**Comment by qiliRedHat** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-ocp-qe-perfscale-router-perf-aws

**Comment by qiliRedHat** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-ocp-qe-perfscale-ci-tests

**Comment by qiliRedHat** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-ocp-qe-perfscale-router-perf-65-gcp

**Comment by qiliRedHat** [comment]
/pj-rehearse periodic periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-ocp-qe-perfscale-node-density-heavy-25-gcp

**Comment by qiliRedHat** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-ocp-qe-perfscale-ci-tests

**Comment by qiliRedHat** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-ocp-qe-perfscale-router-perf-65-gcp

**Comment by qiliRedHat** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-ocp-qe-perfscale-router-perf-gcp periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-ocp-qe-perfscale-node-density-heavy-gcp periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-ocp-qe-perfscale-ci-tests

**Comment by qiliRedHat** [comment]
@paigerube14 periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-ocp-qe-perfscale-ci-tests easily failed for container ready time > 2m in my recent tests.

**Comment by qiliRedHat** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-ocp-qe-perfscale-ci-tests

**Comment by qiliRedHat** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-ocp-qe-perfscale-router-perf-gcp periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-ocp-qe-perfscale-node-density-heavy-gcp periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-ocp-qe-perfscale-ci-tests

**Comment by paigerube14** [comment]
@qiliRedHat is this pr done? I see the tests passed and can give my appropriate comment if you're done/happy with this

**Comment by qiliRedHat** [comment]
@paigerube14 Yes, the tests are done after several retry for the node-density test. I have changed POD_READY_THRESHOLD from 120000ms 180000ms. Let's see how stable the test can pass with this config.

**Pull request merged**